### ISSUE TITLE
Enhancement: Manually updating a collection with Auto-Updating Collections turned on re-starts auto-updation

### ISSUE THREAD

**Issue by pranavAbe** [gilgold/tabox#49]
## Problem Statement 
While the Auto-Updating Collections setting is turned on, if the feature loses the window ID hook for the current collection and stops tracking the collection (say, in a situation where the browser crashed and the window was restored), there is no manual way to resume tracking and updating the collection for the currently open window. The only current way is to close the current window and open the window again through the Tabox collection. 

## Possible Solution
I presume this can be added through the manual **update button** which would be present on the collection that is no longer being actively updated. Clicking this button should hook the current window ID back to the collection and resume auto-updation. 

To summarise, if the current window has already been saved as a collection, but the auto-tracking hook for the current window has been lost, then the **update button** on the collection can be used to re-hook the current window ID back to the collection to resume tracking.

**Comment by gilgold** [comment]
I will see how I can add this option, to basically update + link window to collection. Thanks for another great suggestion!

**Comment by gilgold** [comment]
Added this to the next version (3.5.8). 
This will add a new setting in the settings menu, under Auto Updating Collections: "Click on 'Update' sets 'active'".
Once this is enabled, and auto updating collections is active as well, when you click the update button it will link the collection to the current window.

**Comment by pranavAbe** [comment]
Thanks a lot for putting in the effort to build it in so quick, mate! Will definitely be shouting out Tabox to as many people as I can once these little kinks are sorted out. 

Feel free to close this issue if the feature is set for production in the next update already.

**Comment by gilgold** [comment]
Fixed in v3.5.8 - pending publish in the Chrome and Edge stores
